Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -2507,7 +2508,7 @@ PhysicalParticleContainer::SplitParticles (int lev)
     amrex::Vector<amrex::Vector<ParticleReal>> attr;
     attr.push_back(wp);
     const amrex::Vector<amrex::Vector<int>> attr_int;
-    pctmp_split.AddNParticles(lev,
+    pctmp_split.AddNParticles(lev,   // TODO
```

Comment:
@atmyers @EZoni can you please finalize this? I think using the appropriate particle add function here is all that is needed to complete this.

Reply:
See if you like this way of doing things. I think it makes sense to have this be a `PhysicalParticleContainer`, so we can just call the same `AddNParticles` method.

Reply:
Ah, that makes sense! Good idea.
```suggestion
    pctmp_split.AddNParticles(lev,
```